tasks/deploy_app.yml

```diff
@@ -19,13 +19,39 @@
     that:
       - tc_home is defined
       - tc_home.stat is defined
-      - tc_home.stat.exists
-      - tc_home.stat.isdir 
     quiet: true
 
+- name: Create webapps directory for the application
+  ansible.builtin.file:
+    path: "{{ jws_home }}/webapps"
+    state: directory
+  become: yes
+
+- name: "Add {{ jws_user }} user to root group"
+  ansible.builtin.user:
+    name: "{{ jws_user }}"
+    groups: root
+    append: yes
+
+- name: Check if Tomcat service is running
+  ansible.builtin.systemd_service:
```

Comment:
That's abit cumbersome.. Why not simply directly use:
```
ansible.builtin.systemd_service:
  name: tomcat
  status: started
  ```
  ?

Reply:
for some reason, tomcat is not restarting, so I added an extra task to restart it.

Reply:
I think we need to investigate that and resolve the issue.

Reply:
Fixed it, please look into it.

Reply:
Then why is the ansible.builtin.systemd_service: still here :)

It should be replaced by a populate service + assert ;)